src/index.html

```diff
@@ -30,8 +46,8 @@ <h2>Change the CSS stylesheet below!</h2>
           <input type="radio" id="polyfilledVersion" name="stylesheet" value="polyfilled">
           <label for="polyfilled">Polyfilled</label>
```

Comment:
```suggestion
          <input type="radio" id="polyfilled" name="stylesheet" value="polyfilled">
          <label for="polyfilled">Polyfilled</label>
```

Or the other way around. These currently don't match.